### DISCUSSION TITLE
请求头

### DISCUSSION THREAD

**Discussion by Aurora-Simba** [hooke007/mpv_PlayKit#326] [Q&A 问答]
是否支持请求头 ，如何启用

**Comment by snylonue** [comment]
https://github.com/mpv-player/mpv/blob/c3781b2d6837389c0eb6af6ad0e1998c0df76bc1/player/lua/ytdl_hook.lua#L119C1-L140C1
https://mpv.io/manual/master/#options-http-header-fields
